BridgeAppSDK/SBAInstructionStepViewController.swift

```diff
@@ -109,7 +109,7 @@ open class SBABaseInstructionStepViewController: ORKStepViewController {
         if let image = self.image {
             self.imageView.image = image
         }
-        else {
+        else if self.imageView != nil {
```

Comment:
If you want to change this to possibly be nil, also change the declaration to optional. Currently, this is defined as *expected* to exist.

````
    @IBOutlet public weak var imageView: UIImageView?
````